Added more documentation (#89)
maykinmedia/objects-api: docs/installation/config.md

# Environment configuration reference

> Open Personen can be ran both as a Docker container or directly on a VPS or

Reviewer: ```suggestion
The Objects and Objecttypes API can be ran both as a Docker container or directly on a VPS or
```
